controllers/ibu_controller.go

```diff
@@ -98,8 +100,8 @@ func (r *ImageBasedUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	nextReconcile = doNotRequeue()
 
-	ibu := &ranv1alpha1.ImageBasedUpgrade{}
-	err = r.Get(ctx, req.NamespacedName, ibu)
+	imageBasedUpgrade := &ranv1alpha1.ImageBasedUpgrade{}
```

Comment:
The "ibu" abbreviation is still used in other functions, so it seems inconsistent to change here. Is this change just because "ibu" is now also being used in a for-loop?

Reply:
Yes, the full name is used for the one we are reconciling in the main function